message_processor.go

```diff
@@ -45,6 +48,12 @@ This is a routine to decode the log stream into LogEntry and store it in buffer
 and send the buffer to splunk logger and json logger
 */
 func (mg messageProcessor) consumeLog(lf *logPair) {
+	// Check env variable for json logs
```

Comment:
modify comment - this should come from config not from os variable

Reply:
Comment removed